fs/fs_zenfs.h

```diff
@@ -359,6 +363,9 @@ class ZenFS : public FileSystemWrapper {
     return target()->DeleteDir(ToAuxPath(d), options, dbg);
   }
 
+  IOStatus ForceDeleteDir(const std::string& d, const IOOptions& options,
+                          IODebugContext* dbg);
```

Comment:
I think `IODebugContext* dbg` is not needed as it is not a Rocksdb defined API and also we are not using this anywhere to pass back any information.